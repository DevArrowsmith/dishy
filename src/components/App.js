import '../styles/App.css';
import HomePage from './HomePage/HomePage.js';
import ReviewDishPage from './ReviewDishPage/ReviewDishPage.js';

function App() {
  return (
    <div className="App">
      <HomePage />
    </div>
  );
}

export default App;
