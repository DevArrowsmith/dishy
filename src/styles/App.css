.App {
  min-height: calc(100vh - 80px);
  height: auto;
  width: 100vw;
  padding-top: 80px;
  background-image: url("../assets/text-bg-2.png");
  background-size: 300px auto;
  background-position: top;
  animation: background-scroll 20s linear infinite;
<<<<<<< HEAD
}

@keyframes background-scroll {
  from {
    background-position-x: 0px;
  }
  to {
    background-position-x: 300px;
  }
=======
>>>>>>> c2e5e08b1b42e8b3ddc48abb027b38140190c83f
}

@keyframes background-scroll {
  from {
    background-position-x: 0px;
  }
  to {
    background-position-x: 300px;
  }
}