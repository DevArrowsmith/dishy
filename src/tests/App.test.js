import React from "react";
import { render } from "@testing-library/react";
import App from "../components/App";

describe("App component", () => {
  test("Snapshot test", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
