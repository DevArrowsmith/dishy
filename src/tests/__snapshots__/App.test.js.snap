// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Snapshot test 1`] = `
<DocumentFragment>
  <div
    class="App"
  />
</DocumentFragment>
`;
